### PR TITLE
[Fix #4910] Add an example for Style/AsciiComments into the documentation

### PR DESCRIPTION
**Pull request by jeremyjung**
Add an example for Style/AsciiComments into the documentation.

Note that this code fails `rake_internal_investigation` because the bad example requires putting a non-ascii character into a comment which will cause the investigation to fail.

Before submitting the PR make sure the following are checked:

* [X] Wrote [good commit messages][1].
* [X] Commit message starts with `[Fix #issue-number]` (if the related issue exists).
* [X] Used the same coding conventions as the rest of the project.
* [X] Feature branch is up-to-date with `master` (if not - rebase it).
* [X] Squashed related commits together.
* [X] Added tests.
* [X] Added an entry to the [Changelog](../blob/master/CHANGELOG.md) if the new code introduces user-observable changes. See [changelog entry format](../blob/master/CONTRIBUTING.md#changelog-entry-format).
* [X] All tests(`rake spec`) are passing.
* [ ] The new code doesn't generate RuboCop offenses that are checked by `rake internal_investigation`.
* [X] The PR relates to *only* one subject with a clear title
  and description in grammatically correct, complete sentences.
* [X] Updated cop documentation with `rake generate_cops_documentation` (required only when you've added a new cop or changed the configuration/documentation of an existing cop).

[1]: http://chris.beams.io/posts/git-commit/

### CODE REVIEW

File: lib/rubocop/cop/style/ascii_comments.rb [hunk 1]
```diff
@@ -5,6 +5,12 @@ module Cop
     module Style
       # This cop checks for non-ascii (non-English) characters
       # in comments.
+      # @example
```
Comment (bbatsov) [inline lib/rubocop/cop/style/ascii_comments.rb]: Add a blank line before.

File: lib/rubocop/cop/style/ascii_comments.rb [hunk 2]
```diff
@@ -5,6 +5,12 @@ module Cop
...
+      #   # bad
+      #   # 🤔
```
Comment (bbatsov) [inline lib/rubocop/cop/style/ascii_comments.rb]: A better example would be some text that's partially English and partially something else I guess.

**Comment by bbatsov** [comment]
I notice the documentation in the source and in the manual is different. I assume you wrote it manually instead of using rake generate_cops_documentation.

**Comment by bbatsov** [comment]
That's a fun situation - the build is failing because the cop is raising an offense for the example. Guess you can just disable it locally.

**Comment by jeremyjung** [comment]
I tried disabling cop locally and on a single line.  It passes internal_investigation but adds the "disabling cop" into the cop documentation.

File: lib/rubocop/cop/style/ascii_comments.rb [hunk 3]
```diff
@@ -5,6 +5,13 @@ module Cop
     module Style
       # This cop checks for non-ascii (non-English) characters
       # in comments.
+      #
+      # @example
+      #   # bad
+      #   # Translates from English to 日本語。# rubocop:disable Style/AsciiComments
```
Comment (bbatsov) [inline lib/rubocop/cop/style/ascii_comments.rb]: I guess you can disable this cop for the entire file, so this wouldn't appear in the documentation.

**Pull request merged**